conda/install.py

```diff
@@ -525,28 +542,20 @@ def move_to_trash(prefix, f, tempdir=None):
             if e1.errno != errno.EEXIST:
                 continue
 
-        if tempdir is None:
-            import tempfile
-            trash_dir = tempfile.mkdtemp(dir=trash_dir)
-        else:
-            trash_dir = join(trash_dir, tempdir)
+        trash_dir = tempfile.mkdtemp(dir=trash_dir)
 
         try:
-            try:
-                os.makedirs(join(trash_dir, dirname(f)))
```

Comment:
Ok, how about removing the `prefix, f` parameters, always passing full `path`, and put it in the trash inside a directory that matches the path relative to conda root?

e.g.,

`move_to_trash('c:/miniconda/envs/old_env/lib/site-packages/yaml.dll')`

will move the file to:

`c:/miniconda/pkgs/.trash/tmp19827312/envs/old_env/lib/site-packages/yaml.dll`